src/sagas/index.js

```diff
@@ -1,85 +1,59 @@
 import {
-  put, takeLatest, all, call
+  put, takeLatest, all, call,
 } from 'redux-saga/effects';
 import * as axios from 'axios';
+import { AsyncStorage } from 'react-native';
 import {
-  CREATE_POST,
-  GET_POSTS,
-  POSTS_SUCCESS,
-  POST_NOTES_SUCCESS,
-  DELETE_NOTES_SUCCESS, SAVE_POST, ERROR_GET_POSTS,
-} from '../actions/PostActions';
+  CREATE_USER, ERROR, LOGIN_SUCCESS, LOGIN_USER, LOGOUT_USER,
+} from '../actions/UserActions';
 
-const url = 'http://localhost:8000/notes';
+const url = 'http://localhost:3000/auth';
 
 const combineRequests = {
-  getNotes: () => axios.get(url)
+  login: payload => axios.post(`${url}/login`, payload)
```

Comment:
` axios.post(${url}/login, payload)`
Move **login** to constants. This should be looks like.
` axios.post(${url}/${LOGIN}, payload)`